README.md

```diff
@@ -240,7 +241,7 @@ docker run -d \
     -v $(pwd)/sl/db:/var/lib/postgresql/data \
     --restart always \
     --network="sl-network" \
-    postgres:12.1
+    postgres:13
```

Comment:
we haven't tested postgres:13 yet, as this is run by docker, maybe it's safer to use a tested version?

Reply:
It probably is, I am running with version 13 since 6 months now. I can revert the change if needed